DevelopmentProposals/template.md

```diff
@@ -0,0 +1,63 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. For example:
+Create a graphical tool to plot simulation data.
+
+### Requirements
+
+List the set of requirements that this project must fulfill. Include all constraints this proposal is working within.
+If the list gets too long, consider splitting the project into multiple small projects.
+
+For example:
+
+1. GUI should plot values over time, where values can be joint angles, poses of objects, forces on objects, diagnostic signals, and values from topics.
+1. Up to four values per plot is allowed.
+1. Multiple plots should be supported.
+
+### Architecture
+Include a system architecture diagram.
+This should be a conceptual diagram that describes what components of OpnSim will be utilized or changed, the flow of information, new classes, etc.
+
+### Interfaces
+Describe any new interfaces or modifications to interfaces, function API changes, SDF changes, and GUI changes. These changes can be notional. Describe if/how/when OpenSim users/developers will be affected.
```

Comment:
This overlaps a little with the Backwards Compatibility section. I'm not sure how to resolve that.